subprojects/core/src/main/java/org/gradle/internal/service/scopes/VirtualFileSystemBuildLifecycleListener.java

```diff
@@ -32,9 +32,10 @@ public VirtualFileSystemBuildLifecycleListener(WatchingAwareVirtualFileSystem vi
         this.virtualFileSystem = virtualFileSystem;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Why is this needed now?

Reply:
To get rid of one more compiler warning for using the deprecated `DEPRECATED_VFS_RETENTION_ENABLED_PROPERTY`. It should have been in there before.